content/docs/command-reference/init.md

```diff
@@ -32,67 +32,60 @@ that are hidden from the user. This directory is automatically staged with
 repository. DVC still expects to find the Git repository (will check all
-directories up to the root to find `.git`). This options does not affect any
-config files, `.dvc` directory is created the same way as in the default mode.
-This way multiple DVC projects (including nested ones) could be initialized in a
-single Git repository providing isolation and granular project management.
+directories up to the system root to find `.git/`). This options does not affect
+any config files, `.dvc/` directory is created the same way as in the default
+mode. This way multiple <abbr>DVC projects</abbr> can be initialized in a single
+Git repository, providing isolation between projects.
 
 #### When is this useful?
 
-`--subdir` is mostly used in the scenario of a
-[monorepo](https://en.wikipedia.org/wiki/Monorepo), but also can be used in
-other workflows when such isolation and/or advanced granularity is needed.
+This option is mostly used in the scenario of a
+[monorepo](https://en.wikipedia.org/wiki/Monorepo) (Git repository split into
+several project directories), but can also be used with other patterns when such
+isolation is needed. `dvc init --subdir` mitigates the issues of initializing
+DVC in the Git repository root:
 
-Let's imagine we have an existing Git repository that is split into sub-projects
-(monorepo). In this case `dvc init --subdir` can be run in one or many
-sub-projects to mitigate the issues of initializing in the Git repository root:
+- Repository maintainers might not allow a top level `.dvc/` directory,
+  especially if DVC is being used by several sub-projects (monorepo).
 
-- Repository maintainers might not allow extra `.dvc` top level directory,
-  especially if DVC is being used by a small number of sub-projects.
+- DVC config file, cache directory,
+  [etc.](/doc/user-guide/dvc-files-and-directories) are shared across different
+  sub-projects. This makes it difficult to use different DVC settings,
+  [remote storage](/doc/command-reference/remote) locations, etc.
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- DVC commands like `dvc pull`, `dvc checkout`, `dvc metrics diff`, and plenty
+  others analyze the whole repository to find `dvc.yaml` and `.dvc` files to
+  work with. This may be undesirable and it's inefficient for large repositories
```

Comment:
I'm not sure this is true. For example `dvc repro` only uses ./dvc.yaml by default and needs `-R .` or `-P` to use everything.

Reply:
Hm... It's also repetitive with the next paragraph. I'll update it... ⏳

Reply:
Done.